### ISSUE TITLE
Tree supports often start too high

### ISSUE THREAD

**Issue by mereseydotes** [Ultimaker/Cura#8832]
![PXL_20201125_212947975](https://user-images.githubusercontent.com/74216337/100284156-9b40ad00-2f23-11eb-919f-cce59b296d70.jpg)

**Application version**
4.8.0

**Platform**
Windows 10

**Printer**
CR6- SE

**Reproduction steps**
1. Opened model and sliced with tree supports

**Screenshot(s)**
I added a picture of an example support as it's printing. It may not be clear, as the problem support is on the inside of an existing larger support.

**Actual results**
Sometimes, when a tree support has a wider base, it requires additional folds (?) for adequate support. Sometimes these folds don't start printing until sometime up the model, which means they print in the air and "spaghetti" until they take hold. I will be attaching a file that has a support like this at the back of the model. The "bad" supports tend to grow on the inside of existing tree supports.

**Expected results**
No spaghetti

**Project file**
Link: https://drive.google.com/file/d/1r4bIib9AfLyF-NVtpBHAK2IoNa1Yo6g-/view?usp=sharing

**Log file**
(See https://github.com/Ultimaker/Cura#logging-issues to find the log file to upload, or copy a relevant snippet from it.)

**Additional information**
(Extra information relevant to the issue.)

**Comment by mereseydotes** [comment]
I have also noticed that some models do not get adequate support with support on buildplate only.

**Comment by Ghostkeeper** [comment]
Duplicate of https://github.com/Ultimaker/Cura/issues/4342.
